server/src/main/resources/db/migration/V2_0__Object_Permissions.sql

```diff
@@ -0,0 +1,8 @@
+CREATE TABLE IF NOT EXISTS object_permissions (
+    id INTEGER PRIMARY KEY,
+    object_hash TEXT NOT NULL,
+    grantee_address VARCHAR(44) NOT NULL,
+    created TIMESTAMPTZ NOT NULL
+);
```

Comment:
Should probably try to sneak an `object_byte_size` in here.  A `Bigint` will probably be fine because we will probably put some artificial cap on input payload size (1GB?).  I bet protobuf/grpc won't allow something past a certain size anyhow

Reply:
oh yeah, forgot about that